roles/network_plugin/flannel/handlers/main.yml

```diff
@@ -26,6 +27,11 @@
   service:
     name: docker
     state: restarted
+  when: not is_atomic
+
+- name: Flannel | reload docker (atomic)
+  shell: systemctl stop docker && runc list | awk '!/ID/ {print $1}' | xargs -n 1 -I ID runc kill ID KILL && systemctl start docker
```

Comment:
Would it be better to modify the docker config to set `live-restore = false` before this work?
https://docs.docker.com/engine/admin/live-restore/#enable-the-live-restore-option